runatlantis.io/docs/post-workflow-hooks.md

```diff
@@ -74,4 +76,5 @@ command](custom-workflows.html#custom-run-command).
   * `USER_NAME` - Username of the VCS user running command, ex. `acme-user`. During an autoplan, the user will be the Atlantis API user, ex. `atlantis`.
   * `COMMENT_ARGS` - Any additional flags passed in the comment on the pull request. Flags are separated by commas and
     every character is escaped, ex. `atlantis plan -- arg1 arg2` will result in `COMMENT_ARGS=\a\r\g\1,\a\r\g\2`.
+  * `OUTPUT_STATUS_FILE` - An output file you can use to customize the success or failure status. ex. `echo 'failure' > $OUTPUT_STATUS_FILE`.
```

Comment:
Is this specific to pre and post workflows? Or is it for any status? I ask because the description doesn't specify.

Reply:
It is specific to the pre and post workflows, this is the post-workflow-hooks doc, so I believe this should well scoped. I can add this description if you wish though.